include/simdjson/portability.h

```diff
@@ -4,7 +4,7 @@
 #if defined(__x86_64__) || defined(_M_AMD64)
 #define IS_X86_64 1
 #endif
-#if defined(__aarch64__) || defined(_M_ARM64)
+#if defined(__aarch64__) || defined(_M_ARM64) || defined(__ARM_NEON__) || defined(__ARM_NEON)
 #define IS_ARM64 1
```

Comment:
this should be reworked, it's confusing to say IS_ARM64 when building arm32. Should be renamed to IS_ARM and those arm64 polyfills should be defined for regular arm.

Reply:
This PR was dropped. It was a proof of concept and should be reworked.